### PR TITLE
Fix case sensitive for Argo CD

### PR DESCRIPTION
**Pull request by lbscorpio**
@vikram-redhat There are some case sensitive for Argo CD, minimum version that the change applies to should be 4.7 in my opinion, could you arrange someone to review them? Thanks.

### CODE REVIEW

**Comment by lbscorpio** [comment]
@Preeticp Any update? Thanks.

**Review by reginaelyse — APPROVED** [review]
for 4.7 LGTM!

**Comment by Preeticp** [comment]
Hi @lbscorpio thank you. The labels are likely to change in the upcoming versions, hence we can merge this only in the 4.7 version. Can you please rebase and squash your commits so that this can be merged?

**Comment by lbscorpio** [comment]
@Preeticp Done. But I think that this PR can cherry-pick to 4.8 now, @reginapizza what do you think? Fyi, thanks.

**Comment by reginapizza** [comment]
If the user has the latest GitOps operator installed, the text in the Application Launcher should not change from *OpenShift GitOps* -> *Cluster Argo CD*. It's grabbing that from the Operator code, not devconsole code, so devconsole version won't matter. It would look like this on 4.7, 4.8, 4.9 if the latest gitops operator is installed:
cluster-argo-cd

**Comment by lbscorpio** [comment]
@reginapizza Seems that the Operator code changed since the time of my original commit, I updated the text in the Application Launcher to current words "Cluster Argo CD" in my commit, thanks for your feedback.

File: modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ Red Hat OpenShift GitOps Operator automatically creates a ready-to-use Argo CD i
 .Procedure
 
 . In the *Administrator* perspective of the web console, navigate to *Operators* -> *Installed Operators* to verify that the Red Hat OpenShift GitOps Operator is installed.
-. Navigate to the {rh-app-icon} menu -> *Application Stages* -> *Argo CD*. The login page of the Argo CD UI is displayed in a new window.
+. Navigate to the {rh-app-icon} menu -> *Application Stages* -> *Cluster Argo CD*. The login page of the Argo CD UI is displayed in a new window.
```
Comment (reginapizza) [inline modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc:17]: this is no longer included under `Application Stages` subsection. It's in its own subsection now under `OpenShift GitOps`, as seen in the screenshot I included previously. Sorry for any confusion on that!

**Review by reginaelyse — CHANGES_REQUESTED** [review]

File: modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc (hunk 1 above)
Comment (lbscorpio) [reply modules/logging-in-to-the-argo-cd-instance-by-using-your-openshift-credentials.adoc]: Ok, thanks for the feedback. Updated on my commit.

**Review by reginaelyse — APPROVED** [review]
Thanks, LGTM now :)

**Comment by lbscorpio** [comment]
@Preeticp Any update? Thanks.

**Review by Preeticp — APPROVED** [review]
Thank you @lbscorpio this looks good to me. We would welcome such PRs.

**Pull request merged**